src/test/java/runwar/undertow/WebXMLParserTest.java

```diff
@@ -36,7 +36,7 @@ public void testCommandBoxLuceeWebXML() {
         File webxml = new File(webinf, "lucee4.web.xml");
         boolean ignoreWelcomePages = false;
         boolean ignoreRestMappings = false;
-        WebXMLParser.parseWebXml(webxml, webinf, deploymentInfo, ignoreWelcomePages, ignoreRestMappings);
+        WebXMLParser.parseWebXml(webxml, webinf, deploymentInfo, ignoreWelcomePages, ignoreRestMappings, false);
```

Comment:
Did you actually run the tests to see if this works?  I would expect no, since we also removed the `webinf` argument from the method, but it is still present here in the test.

Reply:
I tested the method in the RunwarConfigurer, all my tests where good, but i did not test the tests, hope that makes sense. do you know the instructions on how to run this tests on runwar? @bdw429s

Reply:
@BalbinoOrtus No, I don't know how to run the tests :)  The previous runwar dev we worked with figured it out, so I'm sure you can too :)  Not that we need it for this pull necessarily-- but it would be nice to document how to run them and see if they are even passing!

Reply:
got it i will get on it to try and make it work